### PR TITLE
Implementing Comments for the Gerrit Change API

### PR DESCRIPTION
**Pull request by anaidoo**
Implementing comments for the Gerrit Change API.  I tried to adhere to the way the other unit tests were written.  Let me know if I should change anything.

### CODE REVIEW

File: src/test/java/com/urswolfer/gerrit/client/rest/http/changes/ChangeApiRestClientTest.java [hunk 1]
```diff
@@ -263,6 +267,27 @@ public void testCheckFix() throws Exception {
 
         EasyMock.verify(gerritRestClient);
     }
+    
+    @Test
+    public void testComments() throws Exception {
+    	  JsonElement jsonElement = EasyMock.createMock(JsonElement.class);
```
Comment (uwolfer) [inline src/test/java/com/urswolfer/gerrit/client/rest/http/changes/ChangeApiRestClientTest.java]: Please use spaces only for indention (this line includes one tab).

File: src/test/java/com/urswolfer/gerrit/client/rest/http/changes/ChangeApiRestClientTest.java [hunk 2]
```diff
@@ -263,6 +267,27 @@ public void testCheckFix() throws Exception {
...
+          GerritRestClient gerritRestClient = new GerritRestClientBuilder()
+              .expectGet("/changes/myProject~master~I8473b95934b5732ac55d26311a706c9c2bde9940/comments", jsonElement)
+              .get();
+
+          TreeMap<String, List<CommentInfo>> expectedCommentInfos = Maps.newTreeMap();
+          CommentsParser commentsParser = EasyMock.createMock(CommentsParser.class);
```
Comment (uwolfer) [inline src/test/java/com/urswolfer/gerrit/client/rest/http/changes/ChangeApiRestClientTest.java]: Please remove trailing spaces on this line.

**Comment by uwolfer** [comment]
@anaidoo: Thanks for you work! Would be nice if you could fix the two minor things.
Btw, your email address used in this commit is not attached to your GitHub account.

**Comment by anaidoo** [comment]
@uwolfer Thanks for pointing out that my email address was wrong.  I have corrected this now and  amended the change.

**Pull request merged**